builder/azure/common/client/config.go

```diff
@@ -76,7 +76,13 @@ type Config struct {
 	// The subscription to use.
 	SubscriptionID string `mapstructure:"subscription_id"`
 
-	authType string
+	// OIDC Request Token is used for GitHub Actions OIDC, this token is used with oidc_request_url to fetch access tokens to Azure
+	// Value in GitHub Actions can be extracted from the `ACTIONS_ID_TOKEN_REQUEST_TOKEN` variable
```

Comment:
I wonder if there's some documentation we can link to for setting up OIDC auth on Github here (or elsewhere)?
IIRC there's some permission/setting to setup in order for this to work, I did it on GCP and I believe this was some setting on the GCP side of things, I would assume something similar exists for Azure?

Reply:
Agreed, that's been added in the newest commit